packages/js/client/src/__tests__/core/plugin-wrapper.spec.ts

```diff
@@ -122,36 +112,28 @@ describe("plugin-wrapper", () => {
       ],
     });
 
-    const queryEnv = await client.query({
+    const queryEnv = await client.invoke({
```

Comment:
Should we also remove the `query` and `mutation` from variable names in tests? (No strong opinion here)

Reply:
yes